src/SampleWebApp/Models/BookDataContext.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using SampleWebApp.Models;
+
+namespace SampleWebApp.Data
+{
+    public class BookDataContext : DbContext
```

Comment:
No, this is also decided by the user. 
Scaffolding allows to generate a controller for a single model per invocation. 

In this case I ran scaffolding twice, and for each of those I chose to create a new DbContext. 

If the user chooses an existing DbContext, scaffolding will only attempt to add the DbSet<> property in the existing DbContext class.